rosette-scala/src/main/scala/coop/rchain/rosette/Location.scala

```diff
@@ -93,4 +91,42 @@ object Location {
 
       case LTLimbo => StoreFail
     }
+
+  def fetch(loc: Location, k: Ctxt, globalEnv: TblObject): Ob =
+    loc.genericType match {
+      case LTCtxtRegister(reg) =>
+        if (reg < k.reg.size) {
```

Comment:
In the C++ implementation `k.reg.size` is the constant `NumberOfCtxtRegs` which is 10.

Reply:
I'd prefer to leave it that way; I suspect the compiler will replace it anyway if the implementation uses a constant for the size.